sim/src/mechanics/queue.rs

```diff
@@ -240,10 +240,9 @@ impl Queue {
     /// If true, there's room and the car must actually start the turn (because the space is
     /// reserved).
     pub fn try_to_reserve_entry(&mut self, car: &Car, force_entry: bool) -> bool {
-        // If lane is already filled, then always return false, even if forced.
-        if self.reserved_length >= self.geom_len {
-            return false;
-        }
+        // If self.reserved_length >= self.geom_len, then the lane is already full. Normally we
```

Comment:
Before, only 2 cars will cross SW-bound at a time, because they're not overfilling the reservation:
![polite](https://user-images.githubusercontent.com/1664407/113770016-81378800-96d6-11eb-9af8-fb21c6388689.gif)
I added this intersection (`53211694`) to the whitelist in the other file, and then 3 will go at a time. Watching it, the behavior is still ridiculous -- the throughput is so low that all of Madison backs up, then Lake Washington gets demolished too.

So I let all of the box-blocking in the world happen:
![allow_blocking](https://user-images.githubusercontent.com/1664407/113770153-a926eb80-96d6-11eb-9955-f10d0ad364f0.gif)
It's not great that vehicles now totally get stuck in the intersection while the north/south stage is active, but in practice for this map, it works and is _much_ better than previously. We still have future work to do to get reasonable throughput, but still not have vehicles block the box this badly.